### ISSUE TITLE
Cochlear nerve hypoplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9638]
**Preferred term label:**
Cochlear nerve hypoplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
300614 
PMID: 25986071   7/7

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added as HP_0034585
